go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/whalebrew/whalebrew
 
-go 1.20
+go 1.21
```

Comment:
Shouldn't we also bump the actions like [here](https://github.com/whalebrew/whalebrew/blob/79b860575946278fd3215e68677d7439424d1cd1/.github/workflows/go.yml#L26)?